lib/callbacks.js

```diff
@@ -48,46 +40,40 @@ const once = (
   return wrap;
 };
 
-const unsafeCallback = (
-  // Extract callback function
-  // It's unsafe: may return null, allows multiple calls
-  args // array, arguments
-  // Returns: function, callback or null
-  // Hint: previous name: `common.cbUnsafe` (deprecated)
```

Comment:
It'd make sense to have them if the function in question actually still existed (we would then make it an alias and write `Deprecated: renamed to..` notice there), but I don't think it's worth having such notes. Because realistically speaking they don't have a limit as they reference already deleted content and will only be arguably useful while cluttering the codebase and docs.

Though I agree that they could be useful but not in the form of code comment. Perhaps if we had a whole changelog for every version and therefore specific separate history for functions names based on that...